west.yml

```diff
@@ -137,7 +137,7 @@ manifest:
       groups:
         - fs
     - name: hal_adi
-      revision: dee9a7b1eff13a9da0560daf8842d61657f9d61e
+      revision: pull/6/head
```

Comment:
I think you are not supposed to make this point to a PR.

Reply:
Pointing to the PR allows us to run CI with the HAL change before merging it.

Reply:
HAL PR has been merged, so this can now be updated to the new SHA

Reply:
Updated.